### ISSUE TITLE
Cannot add multiselect option

### ISSUE THREAD

**Issue by anandmainali** [thedevdojo/voyager#3388]
## Version information

 - Laravel: v5.6.*
 - Voyager: v1.1.*
 - PHP: 7.1.3
 - Database: mysql 
## Description

I am trying to add product size option using multiselect. But i am getting a issue of,

Invalid argument supplied for foreach() (View: /home/helpme/testing/vendor/tcg/voyager/resources/views/bread/browse.blade.php)

## Screenshots
![error](https://user-images.githubusercontent.com/34227482/42550131-811feb74-84ef-11e8-9047-e3b26cd96c32.png)
![select](https://user-images.githubusercontent.com/34227482/42550134-88f16288-84ef-11e8-86c4-b5bbb649c588.png)

**Comment by fletch3555** [comment]
Your `options` should be an array (`[ ]`), not an object (`{ }`)

**Comment by gRoberts84** [comment]
@fletch3555 - This is still an issue.

I have setup an `Select Multiple` field and set the additional details to contain:

```
{"options":{"online":"Online","sms":"SMS","phone":"Phone","postal":"Postal"}}
```

When going to the Add form, the options are available in the dropdown and upon selection, are added to the underlying table as:

```
["online"]
```

When browsing the table, it's expecting an array, but it's returning the raw value. 

The only way I can get around this is to use a mutator, i.e. 

```
    public function getEntryMethodsAttribute() {
        return json_decode($this->attributes['entry_methods'], true);
    }
```

I have also noticed that the default values do not appear to work either (although I cannot find any documentation to support it beyond an empty string)

HTH?

**Comment by gRoberts84** [comment]
Well.... that's annoying.

Seems that issue only affects the browse view. The Edit view expects it to be a string and complains that it is already an array.

**Comment by tommykaira** [comment]
This is an issue for me as well.

![voyager-select-multiple](https://user-images.githubusercontent.com/1387152/44628010-adfe5d00-a97b-11e8-81ea-b2a51fe5c289.png)

![voyager-select-multiple2](https://user-images.githubusercontent.com/1387152/44628060-83f96a80-a97c-11e8-821f-0486fda6f7e0.png)

The value stored in the table is a serialized JSON string.

What I did was I had to cast it to an array type in the model

e.g.

`protected $casts = [
        'options' => 'array',
    ];`
